zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala

```diff
@@ -522,6 +565,9 @@ private[consumer] object Runloop {
     ) extends RebalanceEvent
   }
 
+  // Internal parameters, should not be necessary to tune
+  private final val commandQueueSize: Int = 1024
```

Comment:
The [naming conventions in the scala style guide](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables) dictate that this should be written as:
```suggestion
  private final val CommandQueueSize: Int = 1024
```

Reply:
Not necessarily part of this PR...